liquidity/ui/src/components/Withdraw/Withdraw.tsx

```diff
@@ -39,12 +40,25 @@ export function Withdraw({ isDebtWithdrawal = false }: { isDebtWithdrawal?: bool
   const { minutes, hours, isRunning } = useWithdrawTimer(params.accountId);
   const unlockDate = !isLoadingDate ? accountCollateralUnlockDate : null;
 
-  const maxWithdrawable =
-    network?.preset === 'andromeda' && liquidityPosition
-      ? liquidityPosition.availableCollateral.add(liquidityPosition.availableSystemToken)
-      : isDebtWithdrawal
-        ? liquidityPosition?.availableSystemToken
-        : liquidityPosition?.availableCollateral;
+  const { data: accountCollateral } = useAccountCollateral({
+    accountId: params.accountId,
+    tokenAddress: collateralType?.address,
+  });
+
+  const maxWithdrawable = React.useMemo(() => {
+    if (network?.preset === 'andromeda' && liquidityPosition) {
+      return liquidityPosition.availableCollateral.add(liquidityPosition.availableSystemToken);
+    }
+
+    if (network?.id === 1 && accountCollateral) {
```

Comment:
We can have locked collateral on any chain/deployment

```suggestion
    if (accountCollateral) {
```